package.json

```diff
@@ -56,7 +56,9 @@
     "makelogs": "node scripts/makelogs",
     "spec_to_console": "node scripts/spec_to_console",
     "start": "node scripts/kibana --dev",
-    "start-serverless": "node scripts/kibana --dev --serverless",
+    "start-serverless-es": "node scripts/kibana --dev --serverless=es",
```

Comment:
nit: `start-serverless-*` is a lot to type.  We could consider `serverless-*`.

In my PoC, I used `yarn serverless [type]` and used `type` to pick the `yml` file.

I'm fine with this as-is, but wanted to raise the nit.

Reply:
👍

Reply:
Found https://github.com/dsifford/yarn-completion recently, it works with `serverless-es` but wouldn't work with `serverless es`